content/docs/user-guide/project-structure/index.md

```diff
@@ -5,9 +5,9 @@ project</abbr>, including the internal `.dvc/` directory. From there on, you
 will create and manage different DVC metafiles (below), and populate the
 <abbr>cache</abbr> with data artifacts as you work on your ML experiments.
 
-- `dvc.yaml` files define stages that form the pipeline(s) of a project. All
-  stage-based features such as `dvc params`, `dvc metrics`, and `dvc plots` are
-  specified here.
+- `dvc.yaml` files define stages, parameters, metrics, and plots. Stages form
+  the pipipeline(s) of a project. Parameters, metrics, and plots are used to
+  evaluate and compare project versions.
```

Comment:
Agree this update is warranted but also the way it's explained here may be confusing, since stages can also contain all the other elements, as is probably the most common case (except for `params`). Then again addressing this probably requires revising the entire [dvc.yaml spec page](https://dvc.org/doc/user-guide/project-structure/dvcyaml-files) so we can handle it separately (but let's make an issue?)

Reply:
I reworded, so PTAL. Also added https://github.com/iterative/dvc.org/issues/4251.